umbraco-forms/developer/extending/adding-a-workflowtype.md

```diff
@@ -64,9 +64,39 @@ namespace MyFormsExtensions
 }
 ```
 
-The `Execute()` method gets a `WorkflowExecutionContext` which has properties for the related `Form`, `Record` and `FormState`.  This parameter contains all information related to the workflow.  The `Record` contains all data and meta data submitted by the form. The `Form` references the form the record is from, and `FormState` provides its state.  Other context, such as the current `HttpContext`, if needed can be passed as constructor parameters (For example: the `HttpContext` can be accessed by injecting `IHttpContextAccessor`).
+## Information available to the workflow
 
-You will then need to register this new workflow type as a dependency.
+### Record information
+
+The `Execute()` method gets a `WorkflowExecutionContext` which has properties for the related `Form`, `Record` and `FormState`.  This parameter contains all information related to the workflow.
+
+The `Record` contains all data and meta data submitted by the form.  As shown in the example above, you can iterate over all `RecordField` values in the form. You can also retrieve a specific record field by alias using the following method:
+
+```csharp
+RecordField? recordField = context.Record.GetRecordFieldByAlias("myalias");
+```
+
+Having obtained a reference to a record field, the submitted value can be retrieved via:
+
+```csharp
+var fieldValue = recordField.ValuesAsString();
+```
+
+If the field stores multiple values, they are delimited with a comma. In many cases you can safely split on that delimiter to obtain the individual values. However this can lead to issues if the prevalues being selected also contain commas. If that's a concern, the following extension method is available in `Umbraco.Forms.Core.Extensions` to correctly parse the selected prevalues:
```

Comment:
```suggestion
If the field stores multiple values, they are delimited with a comma. In many cases, you can safely split on that delimiter to obtain the individual values. However, this can lead to issues if the prevalues being selected also contain commas. If that's a concern, the following extension method is available in `Umbraco.Forms.Core.Extensions` to correctly parse the selected prevalues:
```